src/components/Dashboard/FilingHistoryList/filings/ContinuationIn.vue

```diff
@@ -162,12 +175,15 @@ export default class ContinuationIn extends Vue {
     return 'Unknown Name'
   }
 
-  mounted (): void {
+  async mounted (): Promise<void> {
     if (this.tempRegNumber) {
       // auto-expand bootstrap filing
       // assumes this the only filing in the Filing History list (which it should be)
-      this.setPanel(0)
+      await this.toggleFilingHistoryItem(0)
```

Comment:
Here, we do want to wait for toggleFilingHistoryItem() to complete before we enable the textarea. This is a work-around for a bug in the textarea where it sometimes doesn't auto-resize (where its height=0 even when there is text content).